modules/installation-aws-add-local-zone-locations.adoc

```diff
@@ -4,26 +4,26 @@
 
 :_content-type: PROCEDURE
 [id="installation-aws-add-local-zone-locations_{context}"]
-= Opting into AWS Local Zones
+= Opting in to AWS Local Zones
 
 If you plan to create the subnets in AWS Local Zones, you must opt in to each zone group separately.
 
 .Prerequisites
 
 * You have installed the AWS CLI.
-* You have determined into which region you will deploy your {product-title} cluster.
+* You have determined an AWS opt-in Region for where you want to deploy your {product-title} cluster.
 
 .Procedure
 
-. Export a variable to contain the name of the region in which you plan to deploy your {product-title} cluster by running the following command:
+. Export a variable to contain the name of the AWS opt-in Region by running the following command:
 +
 [source,terminal]
 ----
 $ export CLUSTER_REGION="<region_name>" <1>
 ----
-<1> For `<region_name>`, specify a valid AWS region name, such as `us-east-1`.
+<1> For `<region_name>`, specify a valid AWS opt-in Region name, such as `us-east-1`.
 
-. List the zones that are available in your region by running the following command:
+. List the zones that are available in your AWS opt-in Region by running the following command:
```

Comment:
I reached out to an AWS writer and they provided me with a link to https://docs.aws.amazon.com/controltower/latest/userguide/opt-in-region-considerations.html . Do you think the term "AWS opt-in Region" is still OK to you based on this attached article?

Reply:
@dfitzmau I know you can opt-in Regions, but this is out of context here. We are not talking/documenting here about opt-in Regions, we are talking about Local Zones actions (which also can opt-in). So I think you need to keep `AWS Region` instead of `AWS opt-in Region` changes.
thanks

Reply:
Changed five instances of "AWS opt-in Region" to "AWS Region".